src/photos.cs

```diff
@@ -1268,6 +1334,7 @@ interface PHProjectChangeRequest {
 		[Export ("projectExtensionData", ArgumentSemantic.Copy)]
 		NSData ProjectExtensionData { get; set; }
 
+		[Introduced (PlatformName.MacOSX, 10, 13)]
```

Comment:
use the shortcut `[Mac (10,13, onlyOn64: true)]`
easier to remember `onlyOn64: true`

Reply:
Oh cool I didn't know that they meant the same thing haha. So then I can remove it entirely since the header has Mac (10,13) already, correct?

Reply:
yup, no need to dupe them :)